documentation/en/source/pages/templates-spec/builders/builders-outscale.rst

```diff
@@ -176,7 +176,7 @@ If you are using YAML:
 	  installation:
 	    diskSize: 10240
 	  region: eu-west-2
-	  s3bucket: centos-template
+	  bucket: centos-template
```

Comment:
In fact "bucket" it's not used for outscale, we can remove it in examples.
I just look at other field, and add to this, "region" is not the good one, it must be replaced by "zone